Data-Structures/Graph/Graph.js

```diff
@@ -67,6 +67,52 @@ class Graph {
       this.dfs(neighbour, visited, output)
     }
   }
+
+
+  topologicalSortUtil(v, visited, stack) {
+    // Mark the current node as visited.
+    visited[v] = true;
+    let i = 0;
+
+    // Recur for all the vertices adjacent
+    // to thisvertex
+    for (i = 0; i < this.adj[v].length; i++) {
+      if (!visited[this.adj[v][i]]) {
+        this.topologicalSortUtil(this.adj[v][i], visited, stack)
+      }
+    }
+
+    // Push current vertex to stack
+    // which stores result
+    stack.push(v);
+  }
+
+  // The function to do Topological Sort.
+  // It uses recursive topologicalSortUtil()
+  topologicalSort() {
+    let stack = new Array()
+
+    // Mark all the vertices as not visited
+    let visited = new Array(this.V);
+    for (let i = 0; i < this.V; i++) {
+      visited[i] = false;
+    }
+
+    // Call the recursive helper
```

Comment:
You don't need to break this comment across 4 lines. 2 lines or even one line should be fine.